cfg/gtk.cfg

```diff
@@ -1663,9 +1663,16 @@
     <leak-ignore/>
     <noreturn>false</noreturn>
   </function>
+  <!-- gint g_ascii_digit_value(gchar c); -->
   <function name="g_ascii_digit_value">
-    <leak-ignore/>
     <noreturn>false</noreturn>
+    <returnValue type="gint"/>
+    <use-retval/>
+    <leak-ignore/>
+    <arg nr="1" direction="in">
+      <not-uninit/>
+      <not-bool/>
```

Comment:
I am not sure but how about a `<valid>-128:255</valid>`?

Reply:
I was also not sure about the valid values for all these functions. The description for `g_ascii_digit_value` says it takes an ASCII character. So wouldn't the valid values be 0 - 255?

Reply:
Yes. I only looked at the type. Looking at the doc is better. :-)